Liver/Liver.py

```diff
@@ -408,16 +442,33 @@ def onComputeDistanceMapButtonClicked(self):
     parenchymaLabelmapVolumeNode = slicer.mrmlScene.GetFirstNodeByName("ParenchymaLabelMap")
     if not parenchymaLabelmapVolumeNode:
         parenchymaLabelmapVolumeNode = slicer.mrmlScene.AddNewNodeByClass("vtkMRMLLabelMapVolumeNode", "ParenchymaLabelMap")
+    hepaticLabelmapVolumeNode = slicer.mrmlScene.GetFirstNodeByName("HepaticLabelMap")
+    if not hepaticLabelmapVolumeNode:
+      hepaticLabelmapVolumeNode = slicer.mrmlScene.AddNewNodeByClass("vtkMRMLLabelMapVolumeNode", "HepaticLabelMap")
+    portalLabelmapVolumeNode = slicer.mrmlScene.GetFirstNodeByName("PortalLabelMap")
+    if not portalLabelmapVolumeNode:
+      portalLabelmapVolumeNode = slicer.mrmlScene.AddNewNodeByClass("vtkMRMLLabelMapVolumeNode", "PortalLabelMap")
 
     segmentationIds.Initialize()
     segmentationIds.InsertNextValue(tumorSegmentId)
     slicer.modules.segmentations.logic().ExportSegmentsToLabelmapNode(segmentationNode, segmentationIds,
         tumorLabelmapVolumeNode, refVolumeNode)
+
     segmentationIds.Initialize()
     segmentationIds.InsertNextValue(parenchymaSegmentId)
     slicer.modules.segmentations.logic().ExportSegmentsToLabelmapNode(segmentationNode, segmentationIds,
         parenchymaLabelmapVolumeNode, refVolumeNode)
 
+    segmentationIds.Initialize()
+    segmentationIds.InsertNextValue(hepaticSegmentId)
+    slicer.modules.segmentations.logic().ExportSegmentsToLabelmapNode(segmentationNode, segmentationIds,
+        hepaticLabelmapVolumeNode, refVolumeNode)
+
+    segmentationIds.Initialize()
+    segmentationIds.InsertNextValue(portalSegmentId)
+    slicer.modules.segmentations.logic().ExportSegmentsToLabelmapNode(segmentationNode, segmentationIds,
+        portalLabelmapVolumeNode, refVolumeNode)
+
```

Comment:
Do we need the labelmaps? Couldn't we just access the segmentation data without exporting?